### ISSUE TITLE
Orbits cannot be fully unpickled

### ISSUE THREAD

**Issue by Juanlu001** [poliastro/poliastro#762]
🐞 **Problem**

When unpickling orbits, they are not exactly the same.

```
>>> from poliastro.examples import churi
WARNING: AstropyDeprecationWarning: astropy.extern.six will be removed in 4.0, use the six module directly if it is still needed [astropy.extern.six]
>>> import pickle as p
>>> t = p.dumps(churi)
>>> churi.attractor
Sun (☉)
>>> new_churi = p.loads(t)
>>> new_churi
1 x 6 AU x 7.0 deg orbit around Sun (☉) at epoch 2015-11-05 12:00:00.000 (UTC)
>>> churi
1 x 6 AU x 7.0 deg (HCRS) orbit around Sun (☉) at epoch 2015-11-05 12:00:00.000 (UTC)     
>>> churi.frame
<HCRS Frame (obstime=2015-11-05 12:00:00.000)>
>>> new_churi.frame
Traceback (most recent call last):
  File "/home/juanlu/Development/poliastro/poliastro-library/src/poliastro/frames.py", line 281, in get_frame
    frames = _FRAME_MAPPING[attractor]
KeyError: Sun (☉)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/juanlu/Development/poliastro/poliastro-library/src/poliastro/twobody/orbit.py", line 105, in frame
    self._frame = get_frame(self.attractor, self._plane, self.epoch)
  File "/home/juanlu/Development/poliastro/poliastro-library/src/poliastro/frames.py", line 284, in get_frame
    "Frames for orbits around custom bodies are not yet supported"
NotImplementedError: Frames for orbits around custom bodies are not yet supported
```

🖥 **Please paste the output of following commands**

* `pip freeze`

<details>
  
```bash
alabaster==0.7.12
appdirs==1.4.3
asn1crypto==0.24.0
astroid==2.2.5
astropy==3.2.1
astroquery==0.3.9
atomicwrites==1.3.0
attrs==19.1.0
Babel==2.7.0
backcall==0.1.0
beautifulsoup4==4.7.1
black==19.3b0
bleach==3.1.0
blinker==1.4
certifi==2019.6.16
cffi==1.12.3
chardet==3.0.4
Click==7.0
cloudpickle==1.2.1
coverage==4.5.3
cryptography==2.7
cycler==0.10.0
decorator==4.4.0
defusedxml==0.6.0
docutils==0.14
entrypoints==0.3
feedgenerator==1.9
filelock==3.0.12
ghp-import==0.5.5
html5lib==1.0.1
idna==2.8
imagesize==1.1.0
importlib-metadata==0.18
ipykernel==5.1.1
ipython==7.5.0
ipython-genutils==0.2.0
ipywidgets==7.5.1
isort==4.3.20
jedi==0.14.0
jeepney==0.4
Jinja2==2.10.1
jplephem==2.9
json5==0.8.5
jsonschema==3.0.1
jupyter-client==5.3.1
jupyter-core==4.5.0
jupyter-server==0.1.0
jupyterlab==1.0.4
jupyterlab-pygments==0.1.0
jupyterlab-server==1.0.0
keyring==19.0.2
kiwisolver==1.1.0
lazy-object-proxy==1.4.1
llvmlite==0.29.0
Markdown==3.1.1
MarkupSafe==1.1.1
matplotlib==3.1.0
mccabe==0.6.1
mistune==0.8.4
more-itertools==7.0.0
mpmath==1.1.0
nbconvert==5.5.0
nbformat==4.4.0
nbsphinx==0.4.2
nose==1.3.7
notebook==5.7.8
numba==0.44.1
numpy==1.16.4
numpydoc==0.9.1
packaging==19.0
pandas==0.24.2
pandocfilters==1.4.2
parso==0.5.0
pelican==4.0.1
pexpect==4.7.0
pickleshare==0.7.5
pkginfo==1.5.0.1
plotly==4.0.0
pluggy==0.12.0
-e git+git@github.com:Juanlu001/poliastro.git@fb4886a41aab949bc96fac906f1bc6454b22661f#egg=poliastro
prometheus-client==0.7.1
prompt-toolkit==2.0.9
psutil==5.6.3
ptyprocess==0.6.0
py==1.8.0
pycodestyle==2.5.0
pycparser==2.19
pyflakes==2.1.1
Pygments==2.4.2
pylint==2.3.1
pyparsing==2.4.0
PyQt5==5.12.3
PyQt5-sip==4.19.18
PyQtWebEngine==5.12.1
pyrsistent==0.15.2
pytest==5.0.1
pytest-cov==2.5.1
pytest-mpl==0.10
python-dateutil==2.8.0
pytz==2019.1
pyzmq==18.0.1
QtAwesome==0.5.7
qtconsole==4.5.1
QtPy==1.8.0
readme-renderer==24.0
requests==2.22.0
requests-toolbelt==0.9.1
retrying==1.3.3
rope==0.14.0
scipy==1.3.0
SecretStorage==3.1.1
Send2Trash==1.5.0
six==1.12.0
snowballstemmer==1.2.1
soupsieve==1.9.1
Sphinx==2.1.2
sphinx-notfound-page==0.4
sphinx-rtd-theme==0.4.3
sphinxcontrib-applehelp==1.0.1
sphinxcontrib-devhelp==1.0.1
sphinxcontrib-htmlhelp==1.0.2
sphinxcontrib-jsmath==1.0.1
sphinxcontrib-qthelp==1.0.2
sphinxcontrib-serializinghtml==1.1.3
spyder==3.3.5
spyder-kernels==0.5.0
sympy==1.4
terminado==0.8.2
testpath==0.4.2
toml==0.10.0
tornado==6.0.3
tox==3.12.1
tqdm==4.32.2
traitlets==4.3.2
twine==1.13.0
typed-ast==1.4.0
Unidecode==1.1.1
urllib3==1.25.3
virtualenv==16.6.1
voila==0.1.9
wcwidth==0.1.7
webencodings==0.5.1
widgetsnbextension==3.5.0
wrapt==1.11.2
wurlitzer==1.0.3
zipp==0.5.1
```
</details>

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

**Comment by Juanlu001** [comment]
The reason is that bodies should probably be namedtuples, or at least have a sensible `__hash__` method.

**Comment by Juanlu001** [comment]
This was fixed in #763.
